### PR TITLE
Ubuntu-24.04

### PR DESCRIPTION
**Pull request by majst01**
This also use frr-10 in ubuntu and firewall-ubuntu

### CODE REVIEW

File: debian/docker-make.ubuntu.yaml [hunk 1]
```diff
@@ -19,16 +19,16 @@ default-build-args:
...
     tags:
       - ${SEMVER}
       - ${SEMVER_MAJOR_MINOR}
     build-args:
-      - BASE_OS_VERSION=22.04
-      - FRR_VERSION=frr-8
-      - FRR_VERSION_DETAIL=8.5.4-0~ubuntu22.04.1
+      - BASE_OS_VERSION=24.04
+      - FRR_VERSION=frr-10
+      - FRR_VERSION_DETAIL=10.0-0~ubuntu22.04.1
```
Comment (majst01) [inline debian/docker-make.ubuntu.yaml:29]: libyang is already present for ubuntu 24.04 in their debian repo, but not frr

**Review by Gerrit91 — APPROVED** [review]
We need to run this through our integration before merge. Please don't forget the task that copies the image to the release folder in Github Actions.

**Comment by majst01** [comment]
Should we make this a `firewall-4.0-ubuntu` ?

File: debian/docker-make.ubuntu.yaml (hunk 1 above)
Comment (majst01) [reply debian/docker-make.ubuntu.yaml:30]: added hint

**Review by mwennrich — APPROVED** [review]

**Pull request merged**